internal/databases/postgres/bguploader_test.go

```diff
@@ -98,8 +98,8 @@ func TestBackgroundWALUpload(t *testing.T) {
 			tu := testtools.NewMockWalUploader(false, false)
 			fakeASM := asm.NewFakeASM()
 			tu.ArchiveStatusManager = fakeASM
-			bu := postgres.NewBgUploader(a, int32(tt.maxParallelism), int32(tt.maxNumFilesUploaded), tu, false)
 
+			bu := postgres.NewBgUploader(a, int32(tt.maxParallelism), int32(tt.maxNumFilesUploaded), tu, false, false)
```

Comment:
BTW, we would need a test for new functionality.

Reply:
will do

Reply:
added test